mmcmds.c

```diff
@@ -2370,15 +2087,14 @@ void typeProof(long statemNum,
               statemNum /*statemNum*/, 0 /*maxEDepth*/,
               0, /* step; 0 = step 1 */ /*For messages*/
               0,  /*not noDistinct*/
-              /* 3-May-2016 nm */
               2, /* override discouraged-usage statements silently */
-              1 /* Always allow other mathboxes */ /* 5-Aug-2020 nm */
+              1 /* Always allow other mathboxes */
               );
           if (!nmbrLen(nmbrTmpPtr2)) {
-            /* 1-Oct-05 nm Since a proof may not be found for non-standard
+            /* Didn't find syntax proof */

```

Comment:
This detail has been added.  Difficult to verify for me.

Reply:
It was moved from the earlier `bug(236)` line that has since been replaced by a `break` and comment. It's still true that in this branch we didn't find a syntax proof, it's just no longer a bug / impossible situation.